### PR TITLE
Property values proposal as a separate pull request

### PR DESCRIPTION
**Pull request by mfhepp**
This is the property-values proposal turned into a separate contribution, as per

https://github.com/schemaorg/schemaorg/issues/263

It can be used for product properties, EXIF data, and will be useful for the separate automotive extension.

For the full background of the proposal, see https://www.w3.org/wiki/WebSchemas/PropertyValuePairs.

### CODE REVIEW

File: data/schema.rdfa [hunk 1]
```diff
@@ -9167,10 +9174,13 @@ Note that Event uses startDate/endDate instead of startTime/endTime, even when d
     </div>
     <div typeof="rdf:Property" resource="http://schema.org/value">
       <span class="h" property="rdfs:label">value</span>
-      <span property="rdfs:comment">The value of the product characteristic.</span>
+      <span property="rdfs:comment">The value of the quantitative value or property value node. For QuantitativeValue, the recommended type for values is &apos;Number&apos;. For PropertyValue, it can be &apos;Text;&apos;, &apos;Number&apos;, &apos;Boolean&apos;, or &apos;StructuredValue&apos;.</span>
```
Comment (tmarshbing) [inline data/schema.rdfa]: Did you mean to add StructuredValue to the range? It is mentioned in the comment, but not listed in the ranges.

File: data/schema.rdfa [hunk 2]
```diff
@@ -11330,6 +11346,47 @@ postponing for 1.6.
...
+<div typeof="rdf:Property" resource="http://schema.org/additionalProperty">
+    <span class="h" property="rdfs:label">additionalProperty</span>
+    <span property="rdfs:comment">A property-value pair representing an additional characteristics of the entitity, e.g. a product feature or another characteristic for which there is no matching property in schema.org. &lt;br /&gt;&lt;br /&gt;
+	
+	Note: Do not use additionalProperty if there is a specific property for this characteristic readily defined in schema.org. 
+</span>
+    <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/Product">Product</a></span>
+    <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/Place">Place</a></span>
+    <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/QualitativeValue">QualitativeValue</a></span>
+    <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/QualitativeValue">QualitativeValue</a></span>
```
Comment (tmarshbing) [inline data/schema.rdfa]: Did you intend for this to be QuantitativeValue instead?

Comment (mfhepp) [inline data/schema.rdfa]: Oops, yes; thanks for the catch! Will commit a fix shortly.

File: data/schema.rdfa (hunk 1 above)
Comment (mfhepp) [inline data/schema.rdfa]: Yes, thanks - forgot that one! Fix comes in a minute!

**Comment by mfhepp** [comment]
I fixed the description for unitText as requested in the email dated Jan 9, too.

Note that unitText is really important - one one hand, we do want people to use proper UNCEFACT Common Codes when they can, because they are much more reliable and allow unit conversion etc. On the other hand, we want people to publish unit information as text when they cannot do better. Having two separate properties for this maintains backward compatibility with the original GoodRelations model, tools, and data, and reduces the task for a consuming client.

**Comment by mfhepp** [comment]
This pull request is now ready for inclusion in sdo-gozer, following yesterday's discussions.

**Comment by chaals** [comment]
And works for me...

**Comment by danbri** [comment]
Git says I can't merge this automatically, but should be easy enough. Will finish merge and update gozer test site within the week...

**Comment by mfhepp** [comment]
Maybe this is because after you merged the auto proposal, this is no longer from the same codebase.

I can try to update the pull request.

**Comment by mfhepp** [comment]
Hi Dan:

I fixed this, the pull request can now be automatically merged:

https://github.com/schemaorg/schemaorg/pull/377

The cause, afaics, was that the property-values proposal and the automotive proposal added lines in the very same region and git assumed this to be potential conflicts, while they were just sequential additions.


-----------------------------------
martin hepp  http://www.heppnetz.de
mhepp@computer.org          @mfhepp

**Pull request merged**